roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1649,19 +1649,18 @@ def set_proxy_facts(facts):
     """
     if 'common' in facts:
         common = facts['common']
+        if 'no_proxy' in common and isinstance(common['no_proxy'], string_types):
+            common['no_proxy'] = common['no_proxy'].split(",")
+        elif 'no_proxy' not in common:
+            common['no_proxy'] = []
         if 'http_proxy' in common or 'https_proxy' in common:
-            if 'no_proxy' in common and isinstance(common['no_proxy'], string_types):
-                common['no_proxy'] = common['no_proxy'].split(",")
-            elif 'no_proxy' not in common:
-                common['no_proxy'] = []
             if 'generate_no_proxy_hosts' in common and safe_get_bool(common['generate_no_proxy_hosts']):
                 if 'no_proxy_internal_hostnames' in common:
                     common['no_proxy'].extend(common['no_proxy_internal_hostnames'].split(','))
-            # We always add local dns domain and ourselves no matter what
-            common['no_proxy'].append('.' + common['dns_domain'])
-            common['no_proxy'].append('.svc')
-            common['no_proxy'].append(common['hostname'])
-            common['no_proxy'] = ','.join(sort_unique(common['no_proxy']))
+        # We always add local dns domain and ourselves no matter what
```

Comment:
My brain is fuzzy now, are you saying remove the lines under the comment (always add...)? Or move them back where they were?